web/src/actions/chartAction.js

```diff
@@ -76,3 +76,12 @@ export const getSparklines = async () => {
 
 	return chartData;
 };
+
+export const getChartHeaderValues = (pairs) => {
```

Comment:
We already have tickers in the app reducer which is updated by the trades socket but seems it has an issue since it's not updating all fields. I think by fixing that we will have this data out of the box so we no longer need to send a separate request. If fixing the reducer does not get this data, then we can use this request to get the latest ticket. But it's better to update the store here after fetching data and use the state from the store within the application instead of directly using the fetched data.